### ISSUE TITLE
AppImage not working

### ISSUE THREAD

**Issue by jbriard** [afadil/wealthfolio#100]
On Fedora, when I tried to run ./wealthfolio_1.0.11_amd64.AppImage
I get this error:

```
thread 'main' panicked at /home/runner/work/wealthfolio/wealthfolio/src-core/src/db.rs:49:31:
called `Result::unwrap()` on an `Err` value: Os { code: 30, kind: ReadOnlyFilesystem, message: "Read-only file system" }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
With _RUST_BACKTRACE=full_
```
stack backtrace:
   0:     0x55ca43779405 - <std::sys::backtrace::BacktraceLock::print::DisplayBacktrace as core::fmt::Display>::fmt::h1b9dad2a88e955ff
   1:     0x55ca437a5d4b - core::fmt::write::h4b5a1270214bc4a7
   2:     0x55ca4377627f - std::io::Write::write_fmt::hd04af345a50c312d
   3:     0x55ca4377a701 - std::panicking::default_hook::{{closure}}::h96ab15e9936be7ed
   4:     0x55ca4377a3dc - std::panicking::default_hook::h3cacb9c27561ad33
   5:     0x55ca4377ad61 - std::panicking::rust_panic_with_hook::hfe205f6954b2c97b
   6:     0x55ca4377abc7 - std::panicking::begin_panic_handler::{{closure}}::h6cb44b3a50f28c44
   7:     0x55ca437798c9 - std::sys::backtrace::__rust_end_short_backtrace::hf1c1f2a92799bb0e
   8:     0x55ca4377a854 - rust_begin_unwind
   9:     0x55ca42de4c63 - core::panicking::panic_fmt::h3d8fc78294164da7
  10:     0x55ca42de5136 - core::result::unwrap_failed::hfa79a499befff387
  11:     0x55ca42ffc29e - wealthfolio_core::db::init::h04450e881a772533
  12:     0x55ca42ea2875 - wealthfolio_app::main::h29ece2d132729726
  13:     0x55ca42e6a623 - std::sys::backtrace::__rust_begin_short_backtrace::h599f6f430d6f519c
  14:     0x55ca42eb2529 - std::rt::lang_start::{{closure}}::h75ca37da90e8de79
  15:     0x55ca4376e070 - std::rt::lang_start_internal::h5e7c81cecd7f0954
  16:     0x55ca42ea4ed5 - main
  17:     0x7f574d839088 - __libc_start_call_main
  18:     0x7f574d83914b - __libc_start_main@@GLIBC_2.34
  19:     0x55ca42de55f5 - _start
  20:                0x0 - <unknown>
```

**Comment by afadil** [comment]
Should be fixed in v1.1.12. Can you please confirm?

**Comment by jbriard** [comment]
Yes, fixed with the v1.0.12.

Thanks
